Makefile

```diff
@@ -149,6 +148,10 @@ clean-tsconfig:
 	rm -f packages/*/tsconfig.{tsbuildinfo,json}
 	rm -f codemods/*/tsconfig.{tsbuildinfo,json}
 	rm -f eslint/*/tsconfig.{tsbuildinfo,json}
+	# revert files in git
```

Comment:
I don't think we have added any `tsconfig.json` to git. Why we have to revert here? 

@zxbodya If it is for rebasing the `ts` branch, you can split it in a separate commit used by that branch only.

Reply:
there is separate PR #12420  - this is needed when converting other packages, for tsconfigs that can not be generated